src/libs/getPaymentMethodScreenLocation/index.js

```diff
@@ -0,0 +1,11 @@
+/**
+ * We don't need to get the position of the element on native platforms because the popover will be bottom mounted
+ *
+ * @param {Object} nativeEvent
+ * @returns {Object}
+ */
+function getPaymentMethodScreenLocation(nativeEvent) {
```

Comment:
NAB, kind of specific name for this when it doesn't really have anything to do with the payment method screen on its own

Reply:
I'm fine with this but I'd maybe have went with `getBoundingClientRect(element)` and just explain that it doesn't work for native and should only be used in contexts where we don't need this information on native. Maybe @roryabraham has a better idea from his work with the popover stuff.

Reply:
I agree this is overly-specific. Maybe `getPopoverLocationForClickEvent` - to me that makes it more obvious why it returns `0, 0` on native (because Popovers only exist on wide screens).

Which makes me wonder ... is there any reason this function's implementation should be differentiated based on web/native instead of based on wideScreen/narrowScreen?

Reply:
I think yes because otherwise you'd have native code doing `if large screen use getBoundingClientRect()` which is not supported on native e.g. tablets etc.